lib/oli_web/router.ex

```diff
@@ -394,8 +394,14 @@ defmodule OliWeb.Router do
 
     live_session :load_projects,
       on_mount: [OliWeb.LiveSessionPlugs.SetCurrentAuthor, OliWeb.LiveSessionPlugs.SetProject] do
-      live("/:project_id/overview", Projects.OverviewLive)
       live("/:project_id", Projects.OverviewLive)
+      live("/:project_id/overview", Projects.OverviewLive)
+
+      scope "/:project_id/datashop" do
+        pipe_through([:admin])
+
+        live("/analytics", Datashop.AnalyticsLive)
```

Comment:
Let's use `datashop` instead of `analytics` for the route.

Reply:
Also, this route should be in an "admin" only scope.  Otherwise we risk someone that isn't an admin simply typing in the route

Reply:
Makes sense! Just pushed the changes